_posts/2023-08-23-dashboards-usability-snapshot.md

```diff
@@ -0,0 +1,79 @@
+
+Producers and Consumers have a clear idea of how they want to accomplish tasks, and OpenSearch has the capabilities to help users complete their tasks. The participants, however, did not know what features were available or how to do the required actions within OpenSearch Dashboards.
+ 
+Some possible areas for improving Dashboards are: 
+
+* Providing more guidance: Tooltips, auto-complete, and dropdown assistance would ease or eliminate the challenges faced when creating index patterns, setting filters, and searching available options within specific fields.
+* Increasing ease of use: Improving the sorting capabilities and streamlining processes for adding panels to a dashboard and creating index patterns would increase ease of use. Further, providing dashboard templates and widening the click radius (that is, the clickable area around an element) for selecting fields would improve overall usability.
+* Improving alignment with mental models: Adjusting the entry points for editing panels and downloading content and offering security filters would make the dashboards interface more consistent with what users expect. 
+* Enhancing visibility and discoverability of Dashboards features and products: Making dropdowns, filtering capabilities, and visualizations options more accessible and easier to see would improve users' awareness of the available tools.
+* Conducting additional research: Baselines for log analytics using duplicative tools, such as log explorer, and for use cases, such as security analytics, observability, or search, would help to evaluate the usability of various tools and workflows. 
+
+## Conclusion
+
+The findings from the Q1 2023 OpenSearch Dashboards study gave us better insight into what both Producers and Consumers require. We now have better understanding of the current issues, usability gaps, and potential improvements for the overall usability. We are using the key takeaways (provide more user guidance, improve user-friendliness, align better with mental models, make features and tools more visible) can help to prioritize the roadmap of feature improvements while enhancing user experience for Producers and Consumers. 
+
+To learn more about our efforts toward the dashboard vision, look and feel and modernization, please see the Request for Comments (RFC) on GitHub. We encourage you to get involved in our user experience studies. You can sign up at this link: https://amazonmr.au1.qualtrics.com/jfe/form/SV_251DHi1opohhist. 
+
+## References
+
+Sundar, A. 2023. “OpenSearch Partners with Key lime Interactive for Usability Studies: Admin Panel Workflows,” https://opensearch.org/blog/opensearch-partners-with-key-lime-interactive-for-usability-studies-admin-panel-workflows/.
```

Comment:
Is the last comma correct in this citation style, or should it be a period?

Reply:
A period, and I think it should be a hyperlink format. Change made.